src/platform/silabs/wifi/SiWx/WifiInterfaceImpl.cpp

```diff
@@ -790,14 +824,13 @@ sl_status_t WifiInterfaceImpl::TriggerPlatformWifiDisconnection()
 }
 
 #if CHIP_CONFIG_ENABLE_ICD_SERVER
-CHIP_ERROR WifiInterfaceImpl::ConfigurePowerSave(rsi_power_save_profile_mode_t sl_si91x_ble_state,
-                                                 sl_si91x_performance_profile_t sl_si91x_wifi_state, uint32_t listenInterval)
+CHIP_ERROR WifiInterfaceImpl::ConfigurePowerSave(PowerSaveInterface::PowerSaveConfiguration configuration, uint32_t listenInterval)
 {
-    int32_t error = rsi_bt_power_save_profile(sl_si91x_ble_state, RSI_MAX_PSP);
+    int32_t error = rsi_bt_power_save_profile(RSI_SLEEP_MODE_2, RSI_MAX_PSP);
```

Comment:
The default is set to RSI_SLEEP_MODE_2 since there is no power difference between deep sleep and connected sleep modes. 
However, in certain scenarios (e.g., connection retry), it is configured to high power mode (RSI_ACTIVE). Will this cause any issues in such cases?

Reply:
I haven't seen any issue on my tests where the BLE power mode impacted the Wi-Fi side of things. The High Performance requirement on the Wi-Fi seems to overrule the power configuration to force the chip in high power.